build.gradle

```diff
@@ -250,10 +250,10 @@ dependencies {
     implementation "org.unbescape:unbescape:1.1.6.RELEASE"
     implementation "org.slf4j:slf4j-api:1.7.25"
 
-    testImplementation 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.13'
     testImplementation "org.mockito:mockito-core:1.9.5"
-    testImplementation "org.hamcrest:hamcrest-library:1.3"
-    testImplementation "org.hamcrest:hamcrest-core:1.3"
+    testImplementation "org.hamcrest:hamcrest-library:3.0"
```

Comment:
Hamcrest isn’t being split up anymore. Both `org.hamcrest:hamcrest-library` and `org.hamcrest:hamcrest-core` are now basically empty JAR files that both _depend_ on `org.hamcrest:hamcrest` in order to get Maven and Gradle to do the right thing. We should cut out the middleman here.